### DISCUSSION TITLE
Create a New Plugin

### DISCUSSION THREAD

**Discussion by Nsr19** [medusajs/medusa#10527] [Q&A]
Hi Team,

I need to create a new plugin and publish it as a standalone module. Could you please share the steps to accomplish this? Thank you in advance for your assistance!

**Comment by VelestonRato** [comment]
Sure. What plugin are you planning to create ?

You could use the template here to get started : https://github.com/medusajs/medusa-plugin-template

If you need any assistance please feel free to connect back.
